Sources/Views/EpisodePage.swift

```diff
@@ -452,13 +452,20 @@ private func sectionsMenu(episode: Episode, permission: EpisodePermission?) -> N
 
 let divider = Node.hr(attributes: [.class([Class.pf.components.divider])])
 
-func transcriptView(blocks: [Episode.TranscriptBlock], isEpisodeViewable: Bool) -> Node {
+func transcriptView(
+  blocks: [Episode.TranscriptBlock],
+  isEpisodeViewable: Bool,
+  needsExtraPadding: Bool = true
```

Comment:
this is a hack that will be cleaned up soon. the new episode page needs less padding than the old and so I need a way to customizing this.